zerver/tornado/event_queue.py

```diff
@@ -1314,6 +1314,10 @@ def process_notification(notice: Mapping[str, Any]) -> None:
         process_deletion_event(event, user_ids)
     elif event["type"] == "presence":
         process_presence_event(event, cast(List[int], users))
+    elif event["type"] == "cleanup_queue":
```

Comment:
We'll probably want a comment explaining this eventually, but seems like a reasonable approach for now.